src/vm/diagnosticprotocolhelper.h

```diff
@@ -8,15 +8,40 @@
 #ifdef FEATURE_PERFTRACING
 
 #include "common.h"
+#include <diagnosticsprotocol.h>
+
 
 class IpcStream;
 
+// The Diagnostic command set is 0x01
+enum class DumpCommandId : uint8_t
+{
+    // reserved      = 0x00,
+    GenerateCoreDump = 0x01,
+    // future
+};
+
+struct GenerateCoreDumpCommandPayload
+{
+    // The protocol buffer is defined as:
+    //   string - dumpName (UTF16)
+    //   int - dumpType
+    //   int - diagnostics
+    // returns
+    //   ulong - status
+    LPCWSTR dumpName;
+    uint32_t dumpType;
+    uint32_t diagnostics;
+    static const GenerateCoreDumpCommandPayload* TryParse(BYTE* lpBuffer, uint16_t& BufferSize);
+};
+
 class DiagnosticProtocolHelper
 {
 public:
     // IPC event handlers.
 #ifdef FEATURE_PAL
-    static void GenerateCoreDump(IpcStream *pStream); // `dotnet-dump collect`
+    static void GenerateCoreDump(DiagnosticsIpc::IpcMessage& message, IpcStream *pStream); // `dotnet-dump collect`
```

Comment:
I am just curious, what does the backtick mean in c++ comments?

Reply:
I don't think I put that comment there, but I would guess it's the same reason I use them in some of my comments: it's the standard markdown way of indicating code, e.g., `formatted like this`.